packages/shared/src/components/squads/Details.tsx

```diff
@@ -43,11 +46,13 @@ export function SquadDetails({
   onSubmit,
   form,
   createMode = true,
+  onRequestClose,
 }: SquadDetailsProps): ReactElement {
   const { name, handle, description } = form;
+  const [useHandle, setUseHandle] = useState(handle);
```

Comment:
I think it would be best to reserve the verb/prefix `use` when working with custom hooks. WDYT?
```suggestion
  const [shouldUseHandle, setShouldUseHandle] = useState(handle);
```

Reply:
It's not really a should evaluation.
It accepts a incoming handle and then one it actually uses.

Wi'll think about another rename :)

Reply:
Oh, cool. Though it was boolean a boolean from the name. Agreed, let's rename.

Reply:
@sshanzel Went with `activeHandle` does that make more sense?

Reply:
That sounds amazing and intuitive. Great suggestion! 🚢